rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Gazebo      |     11.0.0*    |     11.0.0    |      N/A       |      11.0.0*   |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Ignition    |             Citadel*           |      N/A       |     Citadel*   |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Ogre        |                                      1.10*                       |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| OpenCV      |     4.2.0      |     4.2.0     |     3.4.6*     |      3.2.0     |  4.1.0 / 3.2.0****  |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| OpenSSL     |     1.1.1d     |     1.1.1f    |     1.1.1f     |      1.1.1d    | 1.1.1d / 1.1.1b**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Poco        |     1.9.2      |     1.9.0     |     1.8.0*     |      1.9.0     |        1.9.4        |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Python      |     3.8.0      |     3.8.2     |     3.8.0      |      3.7.3     |  3.8.2 / 3.7.5****  |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Qt          |     5.12.5     |     5.12.3    |     5.10.0     |      5.11.3    | 5.14.1 / 5.12.5**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+|                              |         **Linux only**                                                |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| PCL         |     1.10.0     |     N/A       |     N/A        |      1.9.1     |       1.10.0        |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+|                                **RMW DDS Middleware Providers**                                      |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Connext DDS |                              5.3.1              |               N/A                    |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Cyclone DDS |                                      0.6.x                                             |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Fast-DDS    |                                      2.0.x                                             |
```

Comment:
Again, let's rearrange this list to be:

* Cyclone DDS
* Fast-DDS
* Connext DDS

For Cyclone DDS, we are already at version 0.7.x, so let's change to that here.  For Fast-DDS, we are anticipating landing at least 2.1.x before Galactic (see https://github.com/ros2/ros2/pull/1058), so let's change it to that here.

In all cases we may have to update this again before release.

Reply:
c07dad6